pennylane/ops/op_math/decompositions/controlled_decompositions.py

```diff
@@ -417,12 +418,13 @@ def _mcx_many_workers_condition(num_control_wires, num_work_wires, **__):
 
 
 def _mcx_many_workers_resource(num_control_wires, work_wire_type, **__):
+
+    if work_wire_type == "borrowed":
+        return {ops.Toffoli: 4 * (num_control_wires - 2)}
     return {
-        ops.Toffoli: (
-            4 * (num_control_wires - 2)
-            if work_wire_type == "borrowed"
-            else 2 * (num_control_wires - 2) + 1
-        )
+        qml.TemporaryAND: num_control_wires - 2,
+        adjoint_resource_rep(qml.TemporaryAND): num_control_wires - 2,
+        ops.Toffoli: 1,
```

Comment:
No change in the `"borrowed"` case.

In the `"zeroed"` case, there were `2 * (num_control_wires - 2) + 1` `Toffoli` gates, of which `num_control_wires-2` many are acting on a zeroed work wire as target (so we can replace them by `TemporaryAND`) and `num_control_wires-2` are returning a zeroed work wire as target (so we can replace them by `adjoint(TemporaryAND)`). A single `Toffoli` remains.